src/main/java/org/isf/exatype/rest/ExamTypeController.java

```diff
@@ -89,12 +89,8 @@ public ResponseEntity<ExamTypeDTO> updateExamType(@PathVariable String code, @Re
         }
 
         ExamType examType = examTypeMapper.map2Model(updateExamType);
-        ExamType exType = examTypeBrowserManager.updateExamType(examType);
-        if (exType == null) {
-            throw new OHAPIException(new OHExceptionMessage("Exam Type not updated."));
-        }
-
-        return ResponseEntity.ok(examTypeMapper.map2DTO(exType));
+        ExamType updatedExamType = examTypeBrowserManager.updateExamType(examType);
+        return ResponseEntity.ok(examTypeMapper.map2DTO(updatedExamType));
```

Comment:
what if `updatedExamType` is null?

Reply:
same comment as above

Reply:
yeah, so maybe above we don't need a check like here?